spec/shadow/index.html

```diff
@@ -1123,13 +1123,46 @@
       </section>
 
       <section>
-        <h3>Focus Navigation</h3>
+        <h3>Sequential Focus Navigation</h3>
 
         <p>If a <a>node</a> doesn’t <a data-lt="participates">participate</a> in the <a>document flat tree</a>, the <a>node</a> <strong>must</strong> be skipped from the <a>sequential focus navigation</a>.</p>
 
-        <p>The <a>sequential focus navigation order</a> for a given <a>shadow tree</a> <var>A</var> <strong>must</strong> be inserted into the <a>sequential focus navigation order</a> for the <a>parent tree</a> <var>B</var> as follows:</p>
+        <p>Each <a>shadow root</a> and <a>slot</a> element is a <a>control group owner</a> and
+          forms its <a>control group</a>.
+        </p>
+
+        <p>For a given <a>shadow root</a>, its <a>sequential focus navigation order</a> consists of
+          <a data-lt="focusable area">focusable areas</a> in its <a>shadow tree</a>
+          except descendants of a <a>shadow host</a>.
+          The order is tree order.
+        </p>
+
+        <p>For a given <a>slot</a> element,
+          its <a>sequential focus navigation order</a> consists of
+          <a data-lt="focusable area">focusable areas</a> of its <a>assigned nodes</a>
+          and descendants of the <a>assigned nodes</a>.
+          If a given slot's assigned nodes is empty, its
+          <a>sequential focus navigation order</a> consists of
+          <a data-lt="focusable area">focusable areas</a> of its fallback contents.
```

Comment:
Fallback contents are not well defined.
Also do not forget that <slot> can be a descendant of another <slot>.